src/chrome/content/rules/Amazon-Associates.xml

```diff
@@ -1,4 +1,10 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Non-2xx HTTP code: http://astore.amazon.com/ (200) => https://astore.amazon.com/ (500)
+Fetch error: http://assoc-amazon.ca/ => https://www.assoc-amazon.ca/: (7, 'Failed to connect to assoc-amazon.ca port 80: Connection timed out')
+Fetch error: http://assoc-amazon.fr/ => https://www.assoc-amazon.fr/: (7, 'Failed to connect to assoc-amazon.fr port 80: Connection timed out')
+
```

Comment:
How strange. Both on my direct connection and through Tor (two different circuits) these fail. Perhaps it's a broken DNS load balancing config, and you are hitting correctly configured servers and I am hitting incorrectly configured ones.

I think in the case of hosts that are broken or not depending on your network location, we should choose not to rewrite.